compute/disk_async_replication/main.tf

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+# [START compute_disk_async_replication_parent_tag]
+# [START primary_disk_setup_for_async_replication]
```

Comment:
`primary_disk_setup_for_async_replication` does not follow the region tags prefix standard. Kindly fix this.

Reply:
corrected to `compute_disk_primary `